pyInterface/compareAmplitudeFiles.py

```diff
@@ -43,7 +43,7 @@ def saveTree(path, name, title, lst):
 	for element in lst:
 		arr[0] = element
 		tree.Fill()
-	hist = ROOT.TH1D(name + "_hist", title, 301, -1e-11, 1e-11)
+# 	hist = ROOT.TH1D(name + "_hist", title, 301, -1e-11, 1e-11)
```

Comment:
Ok, then it is probably better to do it like this, which also fixes pylint complaining. Could you add the binning then to the Draw() call and then delete the commented line please?